.gitignore

```diff
@@ -25,6 +26,7 @@ xcuserdata/
 *.moved-aside
 *.xccheckout
 *.xcscmblueprint
+XMLCoder.xcodeproj
```

Comment:
I don't think this file can be removed from the repository or added to .gitignore, since it is required for Carthage support.